File: FrontEnd/Fit-Boy/Assets/Scripts/Quest.cs
public class Quest
{

	public Quest_info info { get; set; }
	public int Xp_reward { get; set; }
    public GPSCoordinate Stop_co { get; set; }
    public GPSCoordinate Start_co { get; set; }
    public int Level;
    public bool crossedRadZone;

	public Quest(Quest_info info, int xp_reward, int level, GPSCoordinate Start_co, GPSCoordinate stop_co)
	{
		this.info = info;
		this.Xp_reward = xp_reward;
		this.Level = level;
        this.Start_co = Start_co;
		this.Stop_co = stop_co;
        crossedRadZone = false;
	}
}

File: Fit_boy/Fit_Boy_Quest_engine java/Quest.java

package Fit_Boy_Quest_engine;

public class Quest 
{
    Quest_info info;
   //String Start_date;
    //String Expiry_date;
   //int Distance;
    int Xp_reward;
    //Coordinate Start_co;
    Coordinate Stop_co;
    int Level;
    boolean Active;
    
    public Quest(Quest_info info, int xp_reward, int level, Coordinate stop_co, boolean active)
    {
        this.info = info;
       // this.Start_date = start_date;
       // this.Expiry_date = expiry_date;
       // this.Distance = distance;
        this.Xp_reward = xp_reward;
        this.Level = level;
       // this.Start_co = start_co;
        this.Stop_co = stop_co;
        this.Active = active;
    }
    
}
